fury/shaders/billboard_spheres_impl.frag

```diff
@@ -0,0 +1,51 @@
+
+/* Calculating the distance between the fragment to the center
+   of the billboard */
+float len = length(point);
+float radius = 1.;
+/* Discarding any fragment not present in a circle of unit radius */
+if(len > radius){
+    discard;
+}
+
+/* Calculating the 3D distance d from the center */
+float d = sqrt(1. - len*len);
+
+/* Calculating the normal as if we had a sphere of radius len*/
+vec3 normalizedPoint = normalize(vec3(point.xy, d));
+
+/* Defining a fixed light direction */
+vec3 direction = normalize(vec3(1., 1., 1.));
```

Comment:
The shader I made for the network demo does not use the vtk lights. It is not critical, but this needs to be addressed at some point. I can help with that, may need to look at the sdf shaders (I believe these were updated to use vtk lights).